src/Cms/FileBlueprint.php

```diff
@@ -57,6 +57,8 @@ public function accept(): array
 	/**
 	 * Returns the list of all accepted MIME types for
 	 * file upload or `*` if all MIME types are allowed
+	 *
+	 * @deprecated 4.2.0 Use `acceptAttribute` instead
```

Comment:
I think the deprecation in 4.2 is fine, however I'd already add a comment for removal so that we don't forget about it:

```suggestion
	 * @deprecated 4.2.0 Use `acceptAttribute` instead
	 * @todo 5.0.0 Remove method
```

I'd say removing in v5 without prior deprecation warning is fine as there was no real use case for external use of this method. What do you think @distantnative?